docs/topics/wasm/wasm-overview.md

```diff
@@ -20,6 +15,12 @@ web applications built with [Compose Multiplatform](https://www.jetbrains.com/lp
 browser in standalone Wasm virtual machines. In the outside-of-browser case,
 [WebAssembly System Interface (WASI)](https://wasi.dev/) provides access to platform APIs, which you can also utilize.
 
+> To run applications built with Kotlin/Wasm in a browser, you need a [browser version](wasm-configuration.md#browser-versions) that supports 
```

Comment:
I find that it's important to remind developers that it's their users who'll need a specific browser in the end, not them :)